Examples/Profile 64-bit App.bat

```diff
@@ -0,0 +1,20 @@
+REM example batch-file to profile a 64-bit .net-application
```

Comment:
Why only .net? This should work with non-managed 64bit apps as well, shouldn't it?